architecture/reusable-controllers.md

```diff
@@ -0,0 +1,61 @@
+---

```

Comment:
I think there is quite some overlap with https://www.betterstimulus.com/architecture/configurable-controllers.html, don't you think?

Now nothing stands against merging those two into one, but then I'd look how to best do that and delete the other one?

Reply:
Oh shit! That's my bad. I honestly didn't even realize the other one was so close.

Not sure what to do about this. I'm confident that you had a goal and an objective. I'm not intimately tied to any of this.